src/main/java/net/sf/jabref/gui/preftabs/FontSelectorDialog.java

```diff
@@ -166,13 +166,13 @@ public FontSelectorDialog(Component comp, Font font) {
 
         familyList = new JList<>(getFontList());
 
-        JPanel familyPanel = createTextFieldAndListPanel(Localization.lang("Font Family"), familyField, familyList);
```

Comment:
The added underscore was just an ugly way to do search and replace in both Java and properties-files. (The translator replaces spaces with underscores, finds the correct property and then replaces the underscores with spaces, so adding it initially doesn't matter.)

The translation changes was primarily finding better wording ("FontSelector" is probably a worse tile compared to "Font selection", most strings do not use title case, so "Font family" instead of "Font Family" etc).

Removed exceptions were not thrown by the methods.

UPPER/LOWER made sense in the old dialog, now LEFT/RIGHT is more intuitive.

Variable names flds, s1, s2 were not that descriptive.

Lambdas

Commented out code was removed.